var/spack/repos/builtin/packages/py-pyside2/package.py

```diff
@@ -69,6 +69,21 @@ class PyPyside2(PythonPackage):
     depends_on("libxslt@1.1.19:", when="+doc", type="build")
     depends_on("py-sphinx", when="+doc", type="build")
 
+    def patch(self):
+        filter_file(
+            r"\=\${shiboken_include_dirs}",
+            ':'.join([
+                r"=${shiboken_include_dirs}",
+                self.spec["qt"]["glx"]["libglx"].prefix.include,
```

Comment:
Didn't even know you could do this. But if pyside2 directly depends on these, it would be better to add explicit dependencies.

Reply:
Causes the build to fail how? Error msg?